.dockerignore

```diff
@@ -0,0 +1,12 @@
+vendor
+node_modules
+*.o
+*.a
+*.so
+*.exe
+*.prof
+/cover
+/cover.html
+/cover.out
+/TAGS
+/.tmp
```

Comment:
I wouldn't actually, it can be a pain, I need to think of an example, but sometimes I have used this docker pattern and wanted to drop into a shell with `docker run -it`, and wanted the .git directory there for various reasons (some tools need this to be a git repo...oh ya, I think it's go get heh).